tools/file_packager.py

```diff
@@ -151,6 +159,8 @@
 
 if (not force) and len(data_files) == 0:
   has_preloaded = False
+if not has_preloaded or jsoutput == None:
```

Comment:
As commented in line 37, this new option is intended to be applicable when preloading and js-output file is specified, so this line force it to false when the said condition is not met. The external metadata filename is derived from the js-output filename, it could not store it separately when the jsoutput file is not stored. When user decide to embed, I suppose he/she intends to include "everything".